Sources/Run/Models/StripeCustomer.swift

```diff
@@ -0,0 +1,81 @@
+import Foundation
+import FluentProvider
+import Fluent
+import Vapor
+
+final public class StripeCustomer: Model {
+    public let storage = Storage()
+    
+    public static let idKey = "stripe_customer_id"
+
+    public var stripe_customer_id: String
+    public var default_paymet_info: String
+    public let user_id: String
+    
+    public var id: Identifier? {
+        get {
+            return Identifier(.string(self.stripe_customer_id), in: nil)
+        }
+        set {
+            self.stripe_customer_id = (newValue?.string)!
```

Comment:
p0: Using force unwrap is very unsafe use
self.charge_id = newValue?.string ?? ""
or
self.charge_id = newValue?.string ?? "n/a"

Reply:
Lilya was copying me... I really like your suggestion here.